mmv1/products/compute/RegionSslPolicy.yaml

```diff
@@ -53,10 +53,12 @@ parameters:
     name: 'region'
     resource: 'Region'
     imports: 'name'
-    required: true
+    required: false
```

Comment:
this can be removed entirely since by default it'll be set to `required: false`

Reply:
removed 👍